--- setup.py.orig	2014-01-20 13:23:11.807193840 -0600
+++ setup.py	2013-02-25 17:15:18.276262574 -0600
@@ -71,7 +71,7 @@
 module1 = Extension(
     'glpk',
     sources = [os.path.join('src',r+'.c') for r in source_roots],
-    define_macros = macros, extra_compile_args=['-m32'], extra_link_args=['-m32'],
+    define_macros = macros, extra_compile_args=['-m64'], extra_link_args=['-m64'],
     library_dirs = libdirs, include_dirs = incdirs,
     libraries = libs, extra_objects = extraobs)
 
